### DISCUSSION TITLE
Why are the text colors in my Ui different from the ones in Patricks Videos?

### DISCUSSION THREAD

**Discussion by brudda0** [smartcontractkit/full-blockchain-solidity-course-py#1649] [Q&A]
Hi, just started but i'm gonna stick around :)
It's not really a problem but something that bugged me since i started with Patrick's course. 
Some of the keywords like types and even the version definition in the second line look different than in the class and also than in screenshots i looked up.
Is this due to a different setting or just because i use a newer version of remix? Couldn't find an option in the settings to change colors of the UI.
Im working on a mac. 
Thanks! Cheers!
<img width="149" alt="Bildschirmfoto 2022-08-15 um 23 48 52" src="https://user-images.githubusercontent.com/111318342/184724483-4875a9e4-4481-4b9f-bb97-2f56e695fdd4.png">
<img width="161" alt="Bildschirmfoto 2022-08-15 um 23 49 01" src="https://user-images.githubusercontent.com/111318342/184724484-f95c7fef-5e6a-4f7f-bf17-0a3e8ea5aab2.png">

**Comment by cromewar** [comment]
Hello @brudda0 
You can setup the color scheme you like the most on the REMIX settings panel, you have a bunch of them there.
